<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>839</width>
    <height>871</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>821</width>
     <height>851</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>819</width>
      <height>849</height>
     </rect>
    </property>
    <widget class="QScrollArea" name="scrollArea_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>620</y>
       <width>791</width>
       <height>180</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>791</width>
       <height>180</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>791</width>
       <height>201</height>
      </size>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>789</width>
        <height>178</height>
       </rect>
      </property>
      <widget class="QTableWidget" name="tableWidget_construcciones">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>361</width>
         <height>161</height>
        </rect>
       </property>
       <column>
        <property name="text">
         <string>Id construcción</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Identificador</string>
        </property>
       </column>
      </widget>
      <widget class="QTableWidget" name="tableWidget_interesados">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>10</y>
         <width>401</width>
         <height>161</height>
        </rect>
       </property>
       <column>
        <property name="text">
         <string>Interesados</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Documento</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Nombre</string>
        </property>
       </column>
      </widget>
     </widget>
    </widget>
    <widget class="QGroupBox" name="Info_complementaria">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>791</width>
       <height>600</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>791</width>
       <height>600</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>791</width>
       <height>541</height>
      </size>
     </property>
     <property name="title">
      <string>Detalle Predio</string>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>70</width>
        <height>16</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>43</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>16</height>
       </size>
      </property>
      <property name="text">
       <string>Id. Operación</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Id_operacion">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>160</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>231</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>221</width>
        <height>20</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>69</width>
        <height>16</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>69</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>69</width>
        <height>16</height>
       </size>
      </property>
      <property name="text">
       <string>Departamento</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Departamento">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>70</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>231</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>231</width>
        <height>20</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>43</width>
        <height>16</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>43</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>43</width>
        <height>16</height>
       </size>
      </property>
      <property name="text">
       <string>Municipio</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Municipio">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>100</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>231</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>231</width>
        <height>20</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>43</width>
        <height>16</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>43</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>43</width>
        <height>16</height>
       </size>
      </property>
      <property name="text">
       <string>Dirección</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_clase_suelo">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>210</y>
        <width>171</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>171</width>
        <height>22</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>171</width>
        <height>22</height>
       </size>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_cat_suelo">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>250</y>
        <width>171</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>171</width>
        <height>22</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>171</width>
        <height>22</height>
       </size>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_dest_econ">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>290</y>
        <width>171</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>171</width>
        <height>22</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>171</width>
        <height>22</height>
       </size>
      </property>
     </widget>
     <widget class="QgsMapCanvas" name="wdt_mapa" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>471</width>
        <height>400</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>471</width>
        <height>400</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>471</width>
        <height>381</height>
       </size>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Direccion">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>130</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>231</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>231</width>
        <height>20</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>210</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>121</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>121</width>
        <height>31</height>
       </size>
      </property>
      <property name="text">
       <string>Clase Suelo</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>250</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>121</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>121</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>Categoría del suelo</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>290</y>
        <width>115</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>115</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>121</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>Destinación Económica</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>30</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Estado</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Estado">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>30</y>
        <width>40</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>221</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>72</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>20</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Número Predial</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_num_predial">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>40</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>231</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>231</width>
        <height>20</height>
       </size>
      </property>
      <property name="maxLength">
       <number>30</number>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>40</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Nº Predial Anterior</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>70</y>
        <width>45</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>45</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Matrícula</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>100</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>20</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Nupre</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>160</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>35</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Avalúo Comercial</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>130</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>30</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Avalúo Catastral</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_num_predial_anterior">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>40</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>231</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>231</width>
        <height>20</height>
       </size>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Matricula">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>70</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>231</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>231</width>
        <height>20</height>
       </size>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Nupre">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>100</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>231</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>231</width>
        <height>20</height>
       </size>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Avaluo_Catastral">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>130</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>231</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>231</width>
        <height>20</height>
       </size>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Avaluo_Comercial">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>160</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>231</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>231</width>
        <height>20</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>483</x>
        <y>334</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>121</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>121</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>Tipo de Predio</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_tipo_predio">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>330</y>
        <width>171</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>171</width>
        <height>22</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>171</width>
        <height>22</height>
       </size>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_cond_predio">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>369</y>
        <width>171</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>171</width>
        <height>22</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>171</width>
        <height>22</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>483</x>
        <y>370</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>121</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>121</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>Condición Predio</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>810</y>
       <width>791</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_VerFichaCons">
        <property name="text">
         <string>Ver Ficha Construcción</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_VerFichaInt">
        <property name="text">
         <string>Ver Ficha Interesados</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Preferred</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>130</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_Grabar">
        <property name="text">
         <string>Grabar</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignRight">
       <widget class="QPushButton" name="pushButton_Cancelar">
        <property name="text">
         <string>Cancelar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsMapCanvas</class>
   <extends>QWidget</extends>
   <header>qgis.gui</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
